pilot/pkg/model/context.go

```diff
@@ -213,12 +213,9 @@ type Proxy struct {
 	XdsResourceGenerator XdsResourceGenerator
 
 	// Active contains the list of watched resources for the proxy, keyed by the DiscoveryRequest short type.
-	Active map[string]*WatchedResource
-
-	// ActiveExperimental contains the list of watched resources for the proxy, keyed by the canonical DiscoveryRequest type.
 	// Note that the key may not be equal to the proper TypeUrl. For example, Envoy types like Cluster will share a single
 	// key for multiple versions.
-	ActiveExperimental map[string]*WatchedResource
+	Active map[string]*WatchedResource
```

Comment:
nit: can you update "keyed by the DiscoveryRequest short type" to "keyed by the DiscoveryRequest typeUrl" while you are here?